src/Ztm.Zcoin.Rpc/ZcoinRpcClient.cs

```diff
@@ -362,70 +388,83 @@ static ushort ToNative(PropertyType type)
 
         class TransactionInfomation
```

Comment:
Once we move `GetExodusTransactionAsync` to public we need to move this one to a dedicated file.